### PR TITLE
Issue 1206 : Prevent i.dgn file upload

### LINKED ISSUE #1206: detect i.dgn and throw file unsupported error
### Description
We support dgn but not idgn. Apparently idgn's extensions are `.i.dgn`, so we need to treat it as special case and return file format not supported.

### Goals
- dgn files should pass
- idgn files should fail

### Tasks
- [x] Add i.dgn as a non-accepted format and run check to prevent uploading
- [ ] Ensure this is also applicable on the backend. 
- [x] Display the i.dgn file error "FILE NOT SUPPORTED"

### PR DESCRIPTION
**Pull request by niblux**
This fixes #1206 : Prevent i.dgn file upload

#### Description
Add a file check to prevent file's with an i.dgn format being accepted.

#### Test cases
- The user will not be able to upload a file with the extension "i.dgn"

### CODE REVIEW

**Comment by niblux** [comment]
Please note some of the change on this commit were from a copy of the latest version of the 'uploadFileComponent'.

File: frontend/routes/teamspaces/components/uploadModelFileDialog/uploadModelFileDialog.component.tsx [hunk 1]
```diff
@@ -68,14 +68,19 @@ export class UploadModelFileDialog extends React.PureComponent<IProps, IState> {
...
 			tag: values.revisionName,
 			desc: values.revisionDesc
 		};
-		uploadModelFile(teamspaceName, projectName, modelId, fileData);
+
+		const modelData = {
+			modelName, modelId
+		};
+
+		uploadModelFile(teamspaceName, projectName, modelData, fileData);
```
Comment (sanmont3drepo) [inline frontend/routes/teamspaces/components/uploadModelFileDialog/uploadModelFileDialog.component.tsx]: I think this breaks the functionality. 

The uploadModelFile  function  that gets called is actually here:
https://github.com/3drepo/3drepo.io/blob/4743a3927cbf4cce777104739fae2be9e3f6f423/frontend/modules/model/model.sagas.ts#L101

And it expects the third parameter to be just the modelId

Comment (niblux) [reply frontend/routes/teamspaces/components/uploadModelFileDialog/uploadModelFileDialog.component.tsx]: Resolved, as error when copying and not merging correctly.

**Comment by niblux** [comment]
I have implemented a back end file format check, which is ready for review. 
Test Cases : When a user uploads a file of an i.dgn file type the upload will return "Format not accepted" error.

**Comment by carmenfan** [comment]
waiting on #1183 to merge first

**Comment by niblux** [comment]
@carmenfan Thank you carmen, i love what you done with the file check. Squeaky clean 👍

**Pull request merged**